pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -701,6 +727,9 @@ const UninitializedCephxKeyCephVersion string = "Uninitialized"
 type LocalCephxStatus struct {
 	// Daemon shows the CephX key status for local Ceph daemons associated with this resources.
 	Daemon CephxStatus `json:"daemon,omitempty"`
```

Comment:
Not related to this PR specifically, but is the `daemon` status a summarized status of all daemons? I thought status would show only individual daemon status for mon, osd, etc

Reply:
In the design doc, most child resources will indicate their daemon key rotation using `status.cephx.daemon`. This doesn't apply to CephCluster (individually-named fields, including `csi`), CephClient (no daemon), or CephBlockPool (no daemon).

Reply:
Another idea is to rename to `status.cephx.rotation`, in case other status related to cephx, but not rotation, is added in the future.